builder/aggregation/hll_sketch_test.go

```diff
@@ -0,0 +1,32 @@
+package aggregation
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHLLSketchBuild(t *testing.T) {
+	hllSketch := NewHLLSketchBuild()
+	hllSketch.SetName("output_name").SetFieldName("metric_name").SetLgK(5).SetTgtHllType("HLL_6")
+
+	// "omitempty" will ignore boolean=false
+	expected := `{"type":"HLLSketchBuild", "name":"output_name", "fieldName": "metric_name", "lgK":5, "tgtHllType":"HLL_6"}`
+
+	hllSketchJson, err := json.Marshal(hllSketch)
+	assert.Nil(t, err)
+	assert.JSONEq(t, expected, string(hllSketchJson))
+}
+
+func TestHLLSketchMerge(t *testing.T) {
+	hllSketch := NewHLLSketchMerge()
+	hllSketch.SetName("output_name").SetFieldName("metric_name").SetLgK(5).SetTgtHllType("HLL_6")
+
+	// "omitempty" will ignore boolean=false
+	expected := `{"type":"HLLSketchMerge", "name":"output_name",  "fieldName": "metric_name", "lgK":5, "tgtHllType":"HLL_6"}`
+
+	hllSketchJson, err := json.Marshal(hllSketch)
+	assert.Nil(t, err)
+	assert.JSONEq(t, expected, string(hllSketchJson))
```

Comment:
probably this applies here https://github.com/golang/go/wiki/CodeReviewComments#initialisms  ( `hllSketchJson` > `hllSketchJSON` )

Reply:
actually this little "issue" is already in codebase: 

```
builder/aggregation/thetasketch_test.go
16:	thetaSketchJson, err := json.Marshal(thetaSketch)
18:	assert.JSONEq(t, expected, string(thetaSketchJson))

```

If you feel to fix that one as well in that PR, I'm fine with it :) thank you!